help/sakura/res/HLP000015.html

```diff
@@ -19,68 +19,68 @@ <h2>開く</h2>
 <br>
 <img alt="OpenFile" src="images/OpenFile.png" /><br>
 <br>
-・<strong>[読み取り専用ファイルとして開く]</strong><br>
+・<span class="bold">[読み取り専用ファイルとして開く]</span><br>
 チェックすると、<a href="HLP000249.html">ビューモード</a>でファイルをオープンします。ファイルを編集することができなくなります。<br>
 ファイルを閲覧目的に使用している場合、大事なファイルをうっかり編集してしまわないようにチェックするといいでしょう。<br>
 <br>
-・<strong>[文字コードセット]</strong><br>
+・<span class="bold">[文字コードセット]</span><br>
 <a href="HLP000271.html">文字コードセット</a>は、以下の中から選択できます。<br>
 開くときの文字コード指定は、開くファイルがどの文字コードセットで書かれているのかを指定します。<br>
 <br>
 また自動判定機能を使えば、文字コードセットの違いを意識せずに、読み込みや編集・保存ができます。<br>
 <div class="li200">
-◆<strong>自動選択</strong><br>
+◆<span class="bold">自動選択</span><br>
 デフォルトです。<br>
 文字コードセットを自動的に認識します。<br>
 ファイルの先頭をある程度まで読み込み、各文字コードセット特有のデータの出現数を調べ、文字コードセットを判断します。<br>
 文字コードの認識を間違えることがあるかもしれません。<br>
 認識処理を行うため、読み込みに時間が掛かります。<br>
 最近のパソコンの動作速度ならば、十分な速度が得られると思います。<br>
-<strong>>Shift_JISのファイルでも、半角カタカナが含まれている場合、EUCと区別がつかないため、よく間違えます。<br>
+<span class="bold">>Shift_JISのファイルでも、半角カタカナが含まれている場合、EUCと区別がつかないため、よく間違えます。<br>
```

Comment:
```suggestion
<span class="bold">Shift_JISのファイルでも、半角カタカナが含まれている場合、EUCと区別がつかないため、よく間違えます。<br>
```

過去の修正ミスにより &gt; が2つ入っています。
同様に修正すべき箇所が他に2つあります。